DNN Platform/Library/Entities/Profile/ProfilePropertyDefinition.cs

```diff
@@ -450,6 +450,16 @@ public bool Visible
             }
         }
 
+
+        [Browsable(false)]
+        [XmlIgnore]
+        public DateTime LastModifiedDate
```

Comment:
There is already a property LastModifiedOnDate in BaseEntityInfo from which ProfilePropertyDefinition driven. There is no need to add new property.